### ISSUE TITLE
Accessibility - iOS - "Exploitation Types" Screen Won't Let a User Open Country with VoiceOver

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#107]
While testing with VoiceOver on the main "Exploitation Types" screen VoceiOver won't let a user open the "Country" screen with VoiceOver as it does for nonscreen reader users. You can hear a click or sound like VoiceOver is activating the button/link, but it doesn't do anything. I'm not sure if VoiceOver is reading it as one item instead to the "Country" and then the list of items for that country.

**Comment by jfc3-dol** [comment]
When using VoiceOver, a user can't go to a given country when it reads the country name. I should be read as a link or a button and be able to go there. With VoiceOver off, I can tap on the country name, and it takes me there while I can tap in a good with or without VoiceOver and go to that screen.

**Comment by jfc3-dol** [comment]
VoiceOver user still can not open the country page when on the main "Exploitation Types" screen as explained above. Someone not using VoiceOver can open the country from this screen.
